glowfic-dl.py

```diff
@@ -56,14 +56,14 @@
 }
 div.post {
```

Comment:
I want to make sure I understand the change here.

You've moved the grey borders between and around posts from being the background of the `posts` div to being the border of the `post` divs. To do this, you needed to move the spacing between posts from being a margin to being the border thickness. However, this means you lose margin collapse, so you manually get it back witbh the `div.post + div.post` rule.

Presumably, based on "make text darkmode-compatible", the goal here is to remove the `background: white` line, so the background gets set by the default css for dark mode.

Is my understanding correct?

Reply:
The cleanup wasn't actually linter-guided; I just spent a couple hours manually cleaning things up before I started working on more-substantial changes, as a way of improving my own understanding of the code and how it flowed, and then went and ran `black` over it in order to keep it formatted in accordance with your patterns up to that point. So I can't point at any tools beyond the one I'm pretty sure you're already using, unfortunately.

Your understanding of the CSS change is correct!

Reply:
Oh ok, fair enough.